src/scrapeLinks.ts

```diff
@@ -27,8 +27,13 @@ const scrapeFromString: (filePath: string, content: string) => string[] = (
         /\[.*?\]\((?:<((?:\(.*?\)|.)*?)>|((?:\(.*?\)|.)*?))(?: ["'].*?["'])?\)/gm,
         (x) => x[2] || x[1]
       );
+      const mdRefLinks = regexMap(
+        content,
+        /\[.*?\]:\s*\n?\s*(.*)/gm,
+        (x) => x[2] || x[1]
+      );
       const hrefLinks = regexMap(content, /href="(.*?)"/gm);
```

Comment:
Per https://github.com/iterative/dvc.org/pull/2567/files#r654707148 (committing)
```suggestion
      const otherLinks = regexMap(content, /(href|link|image)="(.*?)"/gm);
```